src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/HeaderComponent.tsx

```diff
@@ -18,22 +18,52 @@ export function HeaderComponent(props: IHeaderProps) {
     marginBottom: '0',
   };
 
+  var iconPos;
+  var textArr;
+
+  function replaceIcon(element, patern){
+
+    iconPos = element.indexOf(patern);
+
+    if(element.indexOf(patern) !== -1) {
+      header = 
```

Comment:
This works in the case of HeaderComponent, but I'm not sure it will work for all the other components. I will get back to you about that

Reply:
I dont see why it should not work in all cases? But I might be missing something.

For example in Legend.tsx, you are mutating only `props.labelText` (the `textArr` variable is redundant).

Instead of mutating this value, doing `const header = replaceIcon(props.labelText, '{help}');`, and rendering that value instead of `props.labelText` you should get the same result.

The same in ParagraphComponent, the only difference is that the prop is named `text`.